console-frontend/src/components/hydra-console/HydraConsole.jsx

```diff
@@ -305,8 +367,13 @@ class HydraConsole extends React.Component {
             }
             axios.post(this.agentEndpoint + '/send-command', getBody)
               .then( (response) =>  {
+                let outputText = ""
+                if(response.data.length === 0)
+                 outputText = String(JSON.stringify(response.data, this.jsonStringifyReplacer, 8))
```

Comment:
If response.data.length is 0, why would you stringify an empty string?

Reply:
Sure @xadahiya , actually it was because I was rendering the jsx incase its length was >0 so I stringify in case of empty response but thanks for correcting I will make changes accordingly